server/seed.js

```diff
@@ -17,10 +17,22 @@ async function createUser(){
 	})
 };
 
+async function createListing(){
+	await User.destroy({where: {} });
+	await User.create({
```

Comment:
The function is great. The only change we need is instead of destroying and creating users here, we want to destroy and create listings.

For context, what's happening here is Sequelize is doing a SQL query on the listing table, finding all that match the query, which in this case is an empty object, so it will take all of them, then deleting them. This clears the DB to make room for the new sample data. On the next line, we're inserting a new listing with the passed in object properties. 

``` javascript
await Listing.destroy({where: {} });
await Listing.create({ ...

```